internal/provider/indices_resource.go

```diff
@@ -519,6 +519,48 @@ func (r *indicesResource) findAndCreateState(indices []go_marqo.IndexDetail, ind
 	return nil, false
 }
 
+func normalizeIndexSettings(state *IndexResourceModel, newState *IndexResourceModel) {
+	// Handle boolean defaults
+	if state.Settings.TreatUrlsAndPointersAsMedia.IsNull() &&
+		!newState.Settings.TreatUrlsAndPointersAsMedia.ValueBool() {
```

Comment:
If `newState.Settings.TreatUrlsAndPointersAsMedia.ValueBool()` falsey if the boolean value is set to false?